temporal/dynamicconfig/development.yaml

```diff
@@ -31,6 +31,9 @@ history.defaultWorkflowRetryPolicy:
       MaximumIntervalCoefficient: 100.0
       BackoffCoefficient: 2.0
       MaximumAttempts: 0
+log.level:
+  - value: "debug"
+    constraints: {}
```

Comment:
Still seeing info level logs:

```
airbyte-temporal   | {"level":"info","ts":"2021-06-05T09:13:32.886Z","msg":"none","service":"matching","component":"matching-engine","lifecycle":"Started","wf-task-queue-name":"/_sys/temporal-sys-batcher-taskqueue/1","wf-task-queue-type":"Workflow","logging-call-at":"matchingEngine.go:202"}
```

Reply:
Where did you see `log.level` as an option for the dynamic config? I don't see it in https://github.com/temporalio/temporal/blob/master/common/dynamicconfig/constants.go 

I assume there's something that can be provided to the container in the docker-compose file to change how this works.

Reply:
FWIW I'm not that concerned about limiting internal temporal logging. It's noisy on the full view, but it's easy to just pull logs from the api and the scheduler and have a good view of Airbyte as a whole.

Reply:
I was following this documentation about log configuration:
https://docs.temporal.io/docs/server/configuration/#log

Did not realize that the file should follow the go constant file.

Will revert the change.